inland-empire-software-development/main
# #106 - Fix for linting errors

Steps to test:
- `git pull`
- `git checkout fix-eslint-errors`
- `yarn install`
- `yarn lint`
- Make sure no linting errors pop up in the terminal

Submission information [For PR]:

All Submissions:

[  ] The commit message follows our guidelines
[  ] Testing steps have been added to this issue or the PR for this issue
[  ] Submission has been tested in all supported browsers

New Feature Submissions:
[  ] Code gone through a code review by at least one other person
[  ] Code has been locally linted yarn lint
